lib/bootloader_setup.pm

```diff
@@ -486,6 +486,12 @@ sub specific_bootmenu_params {
         $args .= " autoupgrade=1";
     }
 
+    # Boot the system with the debug options if shutdown takes suspiciously long time.
+    # Further actions for saving debug logs are done in 'shutdown/cleanup_before_shutdown' module.
```

Comment:
Do you have a reference, e.g. an external wiki page describing why you add the following parameters?

Reply:
Added.